gensrc/script/doris_builtins_functions.py

```diff
@@ -1661,7 +1662,9 @@
         [['money_format'], 'STRING', ['DECIMAL64'], ''],
         [['money_format'], 'STRING', ['DECIMAL128'], ''],
         [['split_part'], 'STRING', ['STRING', 'STRING', 'INT'], 'ALWAYS_NULLABLE'],
-        [['substring_index'], 'STRING', ['STRING', 'STRING', 'INT'], 'DEPEND_ON_ARGUMENT']
+        [['substring_index'], 'STRING', ['STRING', 'STRING', 'INT'], 'ALWAYS_NULLABLE'],
+        [['substring_index'], 'STRING', ['STRING', 'STRING', 'INT'], 'DEPEND_ON_ARGUMENT'],
```

Comment:
why substring_index is both ALWAYS_NULLABLE and DEPEND_ON_ARGUMENT

Reply:
> why substring_index is both ALWAYS_NULLABLE and DEPEND_ON_ARGUMENT

Bad submission, fixed.